crates/nargo_cli/tests/execution_success/slices/src/main.nr

```diff
@@ -140,7 +141,7 @@ fn merge_slices_mutate(x: Field, y: Field) -> [Field] {
         slice = slice.push_back(x);
     } else {
         slice = slice.push_back(x);
-    }
+    };
```

Comment:
Is this `;` needed?

I'm assuming not since it was not before, but there are some odd parser quirks with `for` sometimes requiring `;`.

Reply:
Not needed, I must have included it on accident. I removed it.